CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## Public contributions
+
+We're so glad you're thinking about contributing to a DOJ USTP open source project! If you feel you have something to add to the project, please submit a pull request. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
+
+We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
```

Comment:
Think this link needs some work. Not sure if we want to link to DOJ code of conduct or use 18F's. Additionally, the link used here goes to an archived GH repo, which in turn points to this instead: https://handbook.tts.gsa.gov/about-us/code-of-conduct/.